ext/project_data.yaml

```diff
@@ -34,22 +34,24 @@ gem_platform_dependencies:
     gem_runtime_dependencies:
       # Pinning versions that require native extensions
       ffi: '~> 1.9.6'
-      win32-dir: '~> 0.4.9'
-      win32-eventlog: '~> 0.6.2'
-      win32-process: '~> 0.7.4'
+      # win32-xxxx gems are pinned due to PUP-6445
+      win32-dir: '= 0.4.9'
+      win32-eventlog: '= 0.6.5'
```

Comment:
Our vendored versions at https://github.com/puppetlabs/puppet-win32-ruby/tree/2.1.x-x64/ruby/lib/ruby/gems/2.1.0/gems are currently:

````yaml
win32-eventlog: '= 0.6.2`
win32-process: '= 0.7.4'
win32-service: '= 0.8.6'
```

We've already covered `win32-service` - what about the differences with the other 2?  Are those versions currently being used in AppVeyor?  If so, that's good enough for me.